docs/supportedsites.md

```diff
@@ -226,6 +226,7 @@
  - **Instagram**
  - **instagram:user**: Instagram user profile
  - **InternetVideoArchive**
+ - **IOL**: iol.pt sites (tvi archive/stream and maisfutebol)
```

Comment:
This file is autogenerated, you can set the `IE_DESC` property in the extractor instead.